the-collab-lab/tcl-36-smart-shopping-list
# App Logo/Home Page Design

## Description

-Added new logo, background gradient, and updated buttons on home screen
-Removed Header component
-Created Learn More modal with bullet point walkthrough of the app
-Created a simple form to join an existing list that will render conditionally based on clicking `Join List`

## Related Issue

closes #48 and #37

## Acceptance Criteria

Make it look good!

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


### After
Landing page:

<img width="1436" alt="Screen Shot 2022-03-02 at 8 35 12 AM" src="https://user-images.githubusercontent.com/78327387/156405838-b5c66961-bc0d-44b2-bf28-439818740f15.png">

After a user clicks Join List:
<img width="1904" alt="Screen Shot 2022-03-03 at 10 22 36 PM" src="https://user-images.githubusercontent.com/78327387/156710799-c7ae946a-5292-4620-a1b1-dbca43b092e2.png">

Learn More modal:
<img width="1905" alt="Screen Shot 2022-03-03 at 10 23 48 PM" src="https://user-images.githubusercontent.com/78327387/156710873-db84498f-3e17-45b1-ba56-a35afe2f45d5.png">



## Testing Steps / QA Criteria
-Pull down `dj-app-logo` to your local environment and `npm start`
-UPDATED: I went ahead and combined the other issue into this PR to keep things simple. Everything is wired up now and should be ready to go. I'm sure there will be adjustments to make for responsiveness - let me know what you find!
